tests/unit/test_messages.py

```diff
@@ -14,25 +12,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+""" """
 
-"""Get OpenAPI spec from FastAPI app and write it as yaml to stdout
-
-    Usage:
-        `scripts/openapi_from_app.py > openapi.yaml`
-"""
-
-
-import sys
+from typing import Dict, Union
 
-import yaml
+import pytest
 
-# Please adapt to package name:
-from interrogation_room.api.main import app
+from .fixtures.kafka_fixtures import event_fixture  # noqa: F401
+from .fixtures.kafka_fixtures import kafka_fixture  # noqa: F401
+from .fixtures.kafka_fixtures import KafkaFixture
 
-# get openapi spec as dict:
-openapi_spec = app.openapi()
 
-# convert to yaml and write to stdout:
-with sys.stdout as stream:
-    print("# This file was autogenerated, please do not modify.", file=stream)
-    openapi_yaml = yaml.safe_dump(openapi_spec, stream)
+@pytest.mark.asyncio
+async def test_incoming_message(
```

Comment:
What is this good for?

Reply:
Leftover sanity check + indirect test if the fixture works. Can be removed.